src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTestData.java

```diff
@@ -0,0 +1,53 @@
+package io.stargate.sgv2.jsonapi.service.operation.filters.table.codecs;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.metadata.schema.ColumnMetadata;
+import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
+import com.datastax.oss.driver.api.core.type.DataType;
+import com.datastax.oss.driver.api.core.type.DataTypes;
+import java.math.BigDecimal;
+import java.util.Optional;
+
+/** Tests data and utility for testing the JSON Codecs see {@link JSONCodecRegistryTest} */
+public class JSONCodecRegistryTestData {
+
+  // A CQL data type we are not going to support soon
+  public final DataType UNSUPPORTED_CQL_DATA_TYPE = DataTypes.COUNTER;
+
+  public final CqlIdentifier TABLE_NAME =
+      CqlIdentifier.fromInternal("table-" + System.currentTimeMillis());
```

Comment:
I guess that's fine, but use of non-reproducible id can not only expose problems (good) but make reproducing said problems difficult (non-reproducible test run; re-run may pass/fail separate from original).